lib/defaults/defaults.go

```diff
@@ -762,23 +751,33 @@ var (
 
 	// FIPSCiphers is a list of supported FIPS compliant SSH ciphers.
 	FIPSCiphers = []string{
-		"aes128-ctr",
-		"aes192-ctr",
+		"aes256-gcm@openssh.com",
 		"aes256-ctr",
+		"aes192-ctr",
```

Comment:
This isn't listed upstream in `x/crypto/ssh`, but seems still supported. I've asked upstream about this.